stations.csv

```diff
@@ -5063,7 +5063,7 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 6670;Miranda de Ebro;miranda-de-ebro;7111200;71112003;42.690554;-2.94091;;ES;Europe/Madrid;f;f;f;t;f;f;ESMEB;;f;;f;7100006;f;375dd5879001acbd84a4683ded6eb0fc;t;ESMEB;t;32081;t;;f;;f;;f;;f;;f;;;;;f;;f;11200;t;;f;ESMEB;f;;f;f;;;;;;;;;;;;ミランダ・デ・エブロ;미란다데에브로;;;Миранда-де-Эбро;;;埃布罗河畔米兰达;urn:trainline:public:nloc:csv6670;
 6671;Murcia;murcia;7161200;71612002;37.974489;-1.131499;;ES;Europe/Madrid;f;f;f;t;f;f;ESMJV;;t;;f;7100052;f;375dd5879001acbd84a4683ded82d0c6;t;ESMJV;t;25291;t;;f;;f;;f;;f;;f;;;;;f;;f;61200;t;;f;ESMJV;f;;f;f;;;;;Murcie;;;;;;;ムルシア;;Murcja;;Мурсия;;;;urn:trainline:public:nloc:csv6671;
 6672;Cordoba;cordoba;7150500;71505008;37.888714;-4.789353;;ES;Europe/Madrid;t;f;f;t;f;f;ESODB;;t;;f;7100038;f;375dd5879001acbd84a4683ded99cf47;t;ESCRO;t;21858;t;;f;;f;;f;;f;;f;;;;;f;;f;50500;t;;f;ESODB;f;;f;f;;;;;Cordoue;Cordova;;;;;;;;;;;;;科尔多瓦;urn:trainline:public:nloc:csv6672;
-6673;Port Aventura;port-aventura;7165411;71654111;41.089251;1.146214;;ES;Europe/Madrid;f;f;f;t;f;f;ESPAV;;f;;f;7100229;f;;f;;f;28731;t;;f;;f;;f;;f;;f;;;;;f;;f;65411;t;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv6673;
+6673;Salou - Port Aventura;salou-port-aventura;7165411;71654111;41.089251;1.146214;;ES;Europe/Madrid;f;f;f;t;f;f;ESPAV;;f;;f;7100229;f;;f;;f;28731;t;;f;;f;;f;;f;;f;;;;;f;;f;65411;t;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv6673;
```

Comment:
Should be an `–` `U+2013 EN DASH`

Reply:
```suggestion
6673;Salou – Port Aventura;salou-port-aventura;7165411;71654111;41.089251;1.146214;;ES;Europe/Madrid;f;f;f;t;f;f;ESPAV;;f;;f;7100229;f;;f;;f;28731;t;;f;;f;;f;;f;;f;;;;;f;;f;65411;t;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv6673;
```

Reply:
Github didn't let me commit your suggestion for some reason but I've fixed it now.